docs/en-us/2.0.0/user_doc/guide/task/sql.md

```diff
@@ -1,22 +1,43 @@
 # SQL
 
-- Drag in the toolbar![PNG](https://analysys.github.io/easyscheduler_docs_cn/images/toolbar_SQL.png)Task node into the drawing board
-- Non-query SQL function: edit non-query SQL task information, select non-query for sql type, as shown in the figure below:
- <p align="center">
-  <img src="/img/sql-en.png" width="80%" />
-</p>
+## Overview
 
-- Query SQL function: Edit and query SQL task information, sql type selection query, select form or attachment to send mail to the specified recipient, as shown in the figure below.
+SQL task, used to connect to database and execute SQL.
 
-<p align="center">
-   <img src="/img/sql-node-en.png" width="80%" />
- </p>
+## create data source
+
+Refer to [Data Source](https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/guide/datasource/introduction.html)
```

Comment:
We should use a related path instead of the full path with a fixable version, in this case, we should use syntax `[Data Source](../datasource/introduction.md)`

Reply:
So as others change, others LGTM